plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -530,7 +530,7 @@ global proc int mayaUsdTranslatorExport (string $parent,
         }
 
         if (stringArrayContains("output", $sectionNames)) {
-            frameLayout -label `getMayaUsdString("kExportFrameOutputLbl")` -collapsable true -collapse false;
+            frameLayout -label `getMayaUsdString("kExportFrameOutputLbl")` -collapsable true -collapse false outputFrameLayout;

```

Comment:
Gave all these frameLayout's names so that from the "Cache to Usd" I can collapse them.